spark-wrapper/spark-2.3/src/main/scala/org/apache/spark/sql/TiAggregationImpl.scala

```diff
@@ -58,13 +89,15 @@ object TiAggregationImpl {
       }
 
       val rewrittenResultExpressions = resultExpressions
-        .map { _ transform rewrite }
+        .map { _ transform avgRewrite }
```

Comment:
one question: should `castedSum` be rewritten if it was generated by `sumRewrite`?